docs/graph_engine/data_spec.rst

```diff
@@ -12,14 +12,86 @@ DeepGNN supports both homogeneous and heterogeneous graphs. Nodes and Edges supp
 Graph Data Format
 *****************
 
-DeepGNN supports two file formats: JSON and TSV.
+DeepGNN supports three file formats: EdgeList, JSON and TSV.
 Users can generate a graph in either format then our pipeline will convert it into binary for training and inference.
 DeepGNN also supports writing custom decoders, see [the decoders file](https://github.com/microsoft/DeepGNN/blob/main/src/python/deepgnn/graph_engine/snark/decoders.py).
 Just inheret the base class Decoder, overwrite the decode function and pass the new decoder as an argument to the converter or dispatcher.
 
-1. `JSON <#json-format>`_: Heterogeneous or homegeneous graph.
+1. `EdgeList <#EdgeList-format>`_: Heterogeneous or homegeneous graph.
 
-2. `TSV <#tsv-format>`_: Homogeneous graph only.
+2. `JSON <#json-format>`_: Heterogeneous or homegeneous graph.
+
+3. `TSV <#tsv-format>`_: Homogeneous graph only.
+
+EdgeList Format
+===============
+
+The EdgeList format,
+* Supports heterogeneous and homegeneous graphs.
+* Nodes and edges are on separate lines, therefore a node and its outgoing edges may be on separate lines.
+* Small files that are fast to create and convert.
+
+`graph.csv` layout,
+
+.. code-block:: text
+	<node info>
+	<edge_1_info>
+	<edge_2_info>
+	...
+
+node_info: node_id,-1,node_type,node_weight,<features>
+edge_info: src,dst,edge_type,edge_weight,<features>
+features[dense]: dtype_name,length,v1,v2,...,dtype_name2,length2,v1,v2,...
+features[sparse with 2 dim coordinates vector]: dtype_name,values.size/coords.shape[1],c1,c2,...,v1,v2,...
+features[sparse with 1 dim coordinates vector]: dtype_name,values.size/0,c1,c2,...,v1,v2,...
```

Comment:
what does `/0` mean?

Reply:
It is a sparse feature where the coordinates vector is only one dimensional.

Reply:
Could you add this to the doc?